Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -140,6 +152,10 @@
       - id: ClothingHandsGlovesInspection # DeltaV - Add inspection gloves for HoP.
       - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
         prob: 0.3
+  # -< DarkSide
```

yamllint

[warning] comment not indented like content

(comments-indentation)